code/datums/mood_events/generic_negative_events.dm

```diff
@@ -310,3 +310,8 @@
 	description = "<span class='warning'>All the fish are dead...</span>\n"
 	mood_change = -3
 	timeout = 90 SECONDS
+
+/datum/mood_event/burnt_wings
+	description = "<span class='boldwarning'>MY PRECIOUS WINGS!!</span>\n"
+	mood_change = -10
+	timeout = 10 MINUTES
```

Comment:
this is a bit, uh, harsh, no? this is basically gonna tank your mood (and thus your sanity) into the gutter unless you immediately drop everything to pursue as many positive moodlets as possible.

Reply:
Correct! It's mechanically reinforcing roleplay in a way I'd like to see more of.

Reply:
bro your organs melted

Reply:
Good.